doc/source/train/doc_code/key_concepts.py

```diff
@@ -37,7 +37,7 @@ def train_fn(config):
             # Local worker rank on the current machine
             "local_rank": session.get_local_rank(),
             # Data
-            "data_shard": next(dataset_shard.iter_batches(batch_format="pandas")),
+            "data_shard": next(iter(dataset_shard.iter_batches(batch_format="pandas"))),
```

Comment:
nit: This seems a bit weird to report, but it also doesn't look like this code block is used anywhere in the docs...